mws/mws.go

```diff
@@ -79,19 +86,45 @@ func ContainsWorkspaceState(workspaceStates []string, searchState string) bool {
-	WorkspaceName          string `json:"workspace_name"`
-	DeploymentName         string `json:"deployment_name"`
-	AwsRegion              string `json:"aws_region"`
-	CredentialsID          string `json:"credentials_id"`
-	StorageConfigurationID string `json:"storage_configuration_id"`
-	CustomerManagedKeyID   string `json:"customer_managed_key_id,omitempty"`
-	NetworkID              string `json:"network_id,omitempty"`
-	IsNoPublicIPEnabled    bool   `json:"is_no_public_ip_enabled,omitempty"`
+	AccountID               string `json:"account_id"`
+	WorkspaceName           string `json:"workspace_name"`
+	DeploymentName          string `json:"deployment_name"`
+	AwsRegion               string `json:"aws_region"`
+	CredentialsID           string `json:"credentials_id"`
+	StorageConfigurationID  string `json:"storage_configuration_id"`
+	CustomerManagedKeyID    string `json:"customer_managed_key_id,omitempty"`
+	NetworkID               string `json:"network_id,omitempty"`
+	IsNoPublicIPEnabled     bool   `json:"is_no_public_ip_enabled,omitempty"`
+	WorkspaceID             int64  `json:"workspace_id,omitempty" tf:"computed"`
+	WorkspaceURL            string `json:"workspace_url,omitempty" tf:"computed"`
+	WorkspaceStatus         string `json:"workspace_status,omitempty" tf:"computed"`
+	WorkspaceStatusMessage  string `json:"workspace_status_message,omitempty" tf:"computed"`
+	PricingTier             string `json:"pricing_tier,omitempty"`
+	PrivateAccessSettingsID string `json:"private_access_settings_id,omitempty"`
+	CreationTime            int64  `json:"creation_time,omitempty" tf:"computed"`
+}
+
+// VPCEndpoint is the object that contains all the information for registering an VPC endpoint
+//Schema From List Customer VPC Endpoint Id API
+type VPCEndpoint struct {
+	VPCEndpointID           string `json:"vpc_endpoint_id,omitempty"`
+	AccountID               string `json:"account_id,omitempty"`
+	VPCEndpointName         string `json:"vpc_endpoint_name"`
+	AwsVPCEndpointID        string `json:"aws_vpc_endpoint_id"`
+	AwsVPCEndpointServiceID string `json:"aws_endpoint_service_id,omitempty"`
+	UseCase                 string `json:"use_case,omitempty"`
+	Region                  string `json:"region"`
+	AwsAccountID            string `json:"aws_account_id,omitempty"`
+	State                   string `json:"state,omitempty" tf:"computed"`
+	CreationTime            int64  `json:"creation_time,omitempty" tf:"computed"`
+}
 
-	WorkspaceID            int64  `json:"workspace_id,omitempty" tf:"computed"`
-	WorkspaceURL           string `json:"workspace_url,omitempty" tf:"computed"`
-	WorkspaceStatus        string `json:"workspace_status,omitempty" tf:"computed"`
-	WorkspaceStatusMessage string `json:"workspace_status_message,omitempty" tf:"computed"`
-	CreationTime           int64  `json:"creation_time,omitempty" tf:"computed"`
+//PAS is the object that contains all the information for creating an PAS
+type PAS struct {
```

Comment:
rename to `PrivateAccessSettings` - codebase should have very descriptive names in order to be supportable.

Reply:
Done